mbed_project/mbed_program.py

```diff
@@ -25,27 +25,25 @@
 class MbedProgram:
     """Represents an Mbed program.
```

Comment:
Sorry I think I missed a few discussions about Mbed Project vs Mbed Program.  I thought that the CLI will mostly only work with projects (ie 1 project = 1 .mbed file) over programs (1 program can contain > 1 projects)? But I could have a completely out of date idea, happy to follow whatever convention you decided on.

Reply:
Ah it looks like I have got it backwards!

Reply:
We decided in one of the stand-ups that we would just deal with programs for now, we don't have the real concept of a "project" yet, which would contain > 1 program. Old tools didn't handle this well either, some things would work by accident but it wasn't well defined in the old tools.

Reply:
That makes sense. I will update the Glossary so I don't forget again. Darn, I have been calling it `project_path` in the other packages 😭

Reply:
Omg, I've even put it in the Glossary, I just couldn't find it because I hadn't put the name in bold 🤦‍♀️